debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/CassandraConnectorConfig.java

```diff
@@ -373,70 +357,72 @@ public CommitLogTransfer getCommitLogTransfer() {
     }
 
     public int maxQueueSize() {
-        return configs.containsKey(MAX_QUEUE_SIZE) ?
-                Integer.parseInt((String) configs.get(MAX_QUEUE_SIZE)) : DEFAULT_MAX_QUEUE_SIZE;
+        return this.getConfig().getInteger(MAX_QUEUE_SIZE);
     }
 
     public int maxBatchSize() {
-        return configs.containsKey(MAX_BATCH_SIZE) ?
-                Integer.parseInt((String) configs.get(MAX_BATCH_SIZE)) : DEFAULT_MAX_BATCH_SIZE;
+        return this.getConfig().getInteger(MAX_BATCH_SIZE);
     }
 
+    /**
+     * Positive integer value that specifies the number of milliseconds the commit log processor should wait during
+     * each iteration for new change events to appear in the queue. Defaults to 1000 milliseconds, or 1 second.
+     */
     public Duration pollIntervalMs() {
-        int ms = configs.containsKey(POLL_INTERVAL_MS) ?
-                Integer.parseInt((String) configs.get(POLL_INTERVAL_MS)) : DEFAULT_POLL_INTERVAL_MS;
+        int ms = this.getConfig().getInteger(POLL_INTERVAL_MS, DEFAULT_POLL_INTERVAL_MS);
         return Duration.ofMillis(ms);
     }
 
     public Duration schemaPollIntervalMs() {
-        int ms = configs.containsKey(SCHEMA_POLL_INTERVAL_MS) ?
-                Integer.parseInt((String) configs.get(SCHEMA_POLL_INTERVAL_MS)) : DEFAULT_SCHEMA_POLL_INTERVAL_MS;
+        int ms = this.getConfig().getInteger(SCHEMA_POLL_INTERVAL_MS);
         return Duration.ofMillis(ms);
     }
 
     public Duration cdcDirPollIntervalMs() {
-        int ms = configs.containsKey(CDC_DIR_POLL_INTERVAL_MS) ?
-                Integer.parseInt((String) configs.get(CDC_DIR_POLL_INTERVAL_MS)) : DEFAULT_CDC_DIR_POLL_INTERVAL_MS;
+        int ms = this.getConfig().getInteger(CDC_DIR_POLL_INTERVAL_MS);
         return Duration.ofMillis(ms);
     }
 
     public Duration snapshotPollIntervalMs() {
-        int ms = configs.containsKey(SNAPSHOT_POLL_INTERVAL_MS) ?
-                Integer.parseInt((String) configs.get(SNAPSHOT_POLL_INTERVAL_MS)) : DEFAULT_SNAPSHOT_POLL_INTERVAL_MS;
+        int ms = this.getConfig().getInteger(SNAPSHOT_POLL_INTERVAL_MS);
         return Duration.ofMillis(ms);
     }
 
     public String[] fieldBlacklist() {
-        String hosts = (String) configs.get(FIELD_BLACKLIST);
+        String hosts = this.getConfig().getString(FIELD_BLACKLIST);
         if (hosts == null) {
             return new String[0];
         }
         return hosts.split(",");
     }
 
+    /**
+     * Whether deletion events should have a subsequent tombstone event (true) or not (false).
+     * It's important to note that in Cassandra, two events with the same key may be updating
+     * different columns of a given table. So this could potentially result in records being lost
+     * during compaction if they haven't been consumed by the consumer yet. In other words, do NOT
+     * set this to true if you have kafka compaction turned on.
+     */
     public boolean tombstonesOnDelete() {
```

Comment:
That's already handled by `CommonConnectorConfig` (see `isEmitTombstoneOnDelete()` and related code).

Reply:
> isEmitTombstoneOnDelete

We'll need to rewrite one here because the default value of `TOMBSTONES_ON_DELETE` in other Debezium Connectors is `true`, but in Cassandra Connector is `false`.

Reply:
Ah, would be nice to align the default then. Is this still something you wanted to do?

Reply:
Make sense! Updated to align the default to be `true` as other debezium connectors. 
But want to double confirm with @jgao54 if it is ok to set the default value of `TOMBSTONES_ON_DELETE` to be `true` ..?

Reply:
Make sense! Updated to align the default to be `true` as other debezium connectors. 
But want to double confirm with @jgao54 if it is ok to set the default value of `TOMBSTONES_ON_DELETE` to be `true` ..?

Reply:
> Ah, would be nice to align the default then. Is this still something you wanted to do?

Make sense! Updated to align the default to be true as other debezium connectors.
But want to double confirm with @jgao54 if it is ok to set the default value of TOMBSTONES_ON_DELETE to be true ..?

Reply:
@bingqinzhou  @gunnarmorling 

Assume my understanding of Kafka tombstone and compaction is correct -- such that when a tombstone is detected, it will remove _all_ events in the log for the key _except for the most recent event with the same key_. If so, I'm a bit concerned with setting it to `true` as default, as it may result in unexpected data losses.

The reason is that the value of a CDC event may not contain all the columns, so the most recent event != the most recent row for a given key.

Reply:
Ah, yes. That's a good point. It seems `false` is the saver default then indeed.

Reply:
In other words, the suggested change here is good AFAICS.